src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -245,10 +240,17 @@ internal static bool ShowCrashErrorReportWindow(DynamoViewModel viewModel, Crash
                         $@"session_start_count=\""0\"" session_clean_close_count=\""0\"" current_session_length=\""0\"" />";
                     }
 
+                    string dynConfig = string.Empty;
+                    string dynName = viewModel?.Model.CurrentWorkspace.Name;
+                    if (!string.IsNullOrEmpty(dynName))
+                    {
+                        dynConfig = $"/DWG {dynName}";
```

Comment:
Send the name of the Dynamo file.
https://wiki.autodesk.com/pages/viewpage.action?spaceKey=NCERS&title=CER+Component+Integration+Guidance+-+6.1.x#CERComponentIntegrationGuidance6.1.x-DWG